homeassistant/components/alexa/handlers.py

```diff
@@ -1523,3 +1524,30 @@ def temperature_from_object(hass, temp_obj, interval=False):
     )
 
     return directive.response()
+
+
+@HANDLERS.register(("Alexa.CameraStreamController", "InitializeCameraStreams"))
+async def async_api_initialize_camera_stream(hass, config, directive, context):
+    """Process a InitializeCameraStreams request."""
+    entity = directive.entity
+    camera_component = hass.data.get(camera.DOMAIN)
+    camera_entity = camera_component.get_entity(entity.entity_id)
+    stream_source = await camera.async_request_stream(
+        hass, camera_entity.entity_id, fmt="hls"
+    )
+    payload = {
+        "cameraStreams": [
+            {
+                "uri": f"{config.camera_stream_url}{stream_source}",
+                "protocol": "RTSP",
+                "resolution": {"width": 1280, "height": 720},
+                "authorizationType": "NONE",
+                "videoCodec": "H264",
+                "audioCodec": "AAC",
```

Comment:
These too, we don't know this do we ?

Reply:
There is currently no way to determine the frame size of the video for each streaming device. The stream component uses H264 for video, and AAC for audio when using HLS transport. After thorough testing on available echo devices, the device will display the stream regardless of the requested frame size.